podvm-mkosi/mkosi.skeleton/usr/lib/systemd/system/kata-agent.service.d/10-override.conf

```diff
@@ -1,10 +1,8 @@
 # On a read-only fs the kata-agent config is created in /run/peerpod, since it contains
 # a parameter that can be set at pod creation time.
-[Unit]
-ConditionKernelCommandLine=
```

Comment:
Was this removed by the original unit? Otherwise lgtm, thanks.

Reply:
ah, no this was not intentionally removed. but it doesn't do anything, does it? the kata-agent unit doesn't have this set.

Reply:
I think it had some condition at some point, but couldn't find one in current main.